src/util/extent.js

```diff
@@ -128,9 +128,9 @@ export default function() {
 
     extents.include = function(x) {
         if (!arguments.length) {
-            return extraPoint;
+            return extraPoints;
         }
-        extraPoint = x;
+        extraPoints = x;
```

Comment:
I thought this would make the interface even smoother, if you are calling include twice on two separate arrays.